x-pack/plugins/transform/public/app/hooks/use_start_transform.ts

```diff
@@ -6,23 +6,19 @@
 
 import { i18n } from '@kbn/i18n';
 
-import { TransformEndpointRequest, TransformEndpointResult } from '../../../common';
+import type { StartTransformsRequestSchema } from '../../../common/api_schemas/start_transforms';
```

Comment:
i bet there are a lot of places in the ML code base where we could use type only imports.

Reply:
I added a related note to this issue: https://github.com/elastic/kibana/issues/77041